src/types/Activity.ts

```diff
@@ -0,0 +1,19 @@
+import { components } from 'src/api/types/generated-schema';
+
+export type Activity = Omit<
+  components['schemas']['AdminActivityPayload'],
+  'createdBy' | 'createdTime' | 'isVerified' | 'modifiedBy' | 'modifiedTime'
+> & {
+  createdBy?: number;
+  createdTime?: string;
+  isVerified?: boolean;
+  media: components['schemas']['ActivityMediaFilePayload'][];
+  modifiedBy?: number;
+  modifiedTime?: string;
+};
```

Comment:
Doing this for now to simplify things, will separate admin & non-admin types in a future PR.